beancount_dkb/ec.py

```diff
@@ -24,10 +24,12 @@
 
 
 class ECImporter(importer.ImporterProtocol):
-    def __init__(self, iban, account, currency='EUR', file_encoding='utf-8'):
+    def __init__(self, iban, account, currency='EUR', file_encoding='utf-8',
+                 meta_code=None):
         self.account = account
         self.currency = currency
         self.file_encoding = file_encoding
+        self.meta_code = meta_code
```

Comment:
I wonder if it would make sense to update the `README` with an example where `meta_code` is set.

Reply:
I can do that. Patience is needed though, because I'm quite busy right now :)

Reply:
Turns out I'm apparently not that busy enough. Added some doc. Feedback welcome.